src/test/resources/release-notes-file.md

```diff
@@ -0,0 +1,11 @@
+### New features
+
+* [JIRA-TICKET](jira-link): Some feature
```

Comment:
For the record, I was going for something like this: https://github.com/apache/subversion/blob/trunk/CHANGES